chargeOclock/Source/Other/NetworkAPI.swift

```diff
@@ -55,6 +58,29 @@ class NetworkAPI {
 			}.resume()
 	}
 
+	fileprivate func performDeleteRequest(to uri: String, completion: @escaping NetworkCompletion) {
```

Comment:
If I understand correctly, we first need `performRequest` and only after that we can implement `performPostRequest`. And `performPostRequest` is intended for creation of client. Am I correct? Because I was thinking to proceed with create client, but I'm not sure if that's the thing to do right now?